src/structures/interfaces/InteractionResponses.js

```diff
@@ -161,7 +161,8 @@ class InteractionResponses {
    * @param {string|MessagePayload|InteractionReplyOptions} options The options for the reply
    * @returns {Promise<Message|APIMessage>}
    */
-  followUp(options) {
+  async followUp(options) {
+    if (!this.deferred && !this.replied) throw new Error('INTERACTION_NOT_REPLIED');
```

Comment:
Since ESLint is dumb and doesn't see that this is intentional to do promise rejections and not just to `await`...
```suggestion
  followUp(options) {
    if (!this.deferred && !this.replied) return Promise.reject(new Error('INTERACTION_NOT_REPLIED'));
```

Fixes `Async method 'followUp' has no 'await' expression` warning.